doc/apimanager-reference.md

```diff
@@ -731,25 +731,21 @@ The available configurable secrets are:
 
 For Mysql:
 
-| **Field** | **Description** | **Default value** |
-| --- | --- | --- |
-| URL | URL of the Porta database. Format: `mysql2://<AdminUser>:<AdminPassword>@<DatabaseHost>/<DatabaseName>`, where `<AdminUser>` must be an already existing user in the external database with full permissions on the specified `<DatabaseName>` logical database and `<DatabaseName>` must be an already existing logical database in the external database.| Mandatory when the instance is managed externally. A default is only set when database is managed internally.<br/>When managed internally:<br/>`mysql2://root:<AutogeneratedValue>@system-mysql/mysql`.|
-| DB_USER | Not used by 3scale components. Only used when the database is managed internally to create a new user granted with superuser permissions for the database specified in the `URL` field. | `mysql` |
-| DB_PASSWORD | Not used by 3scale components. Only used when the database is managed internally to create a new user granted with superuser permissions for the database specified in the `URL` field. | Autogenerated value |
+| **Field** | **Description**                                                                                                                                                                                                                                                                                                                                   | **Default value** |
+| --- |---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------| --- |
+| URL | URL of the Porta database. Format: `mysql2://<User>:<Password>@<DatabaseHost>/<DatabaseName>`, where `<User>` must be an already existing user in the external database with required permissions on the specified `<DatabaseName>` logical database and `<DatabaseName>` must be an already existing logical database in the external database. | Mandatory when the instance is managed externally. A default is only set when database is managed internally.<br/>When managed internally:<br/>`mysql2://root:<AutogeneratedValue>@system-mysql/mysql`.|
```

Comment:
```suggestion
| URL | URL of the Porta database. Format: `mysql2://<User>:<Password>@<DatabaseHost>/<DatabaseName>`, where `<User>` must be an already existing user in the external database with required permissions on the specified `<DatabaseName>` logical database and `<DatabaseName>` must be an already existing logical database in the external database. | Mandatory when the instance is managed externally. A default is only set when database is managed internally.<br/>When managed internally:<br/>`mysql2://mysql:<AutogeneratedValue>@system-mysql/mysql`.|
```
Need to change default username from `root` to `mysql` in doc

Reply:
updated. Thank you